CodeStyleExercise/ColorsViewController.swift

```diff
@@ -8,55 +8,52 @@
 
 import UIKit
 
-class ColorsViewController : UIViewController, UICollectionViewDataSource, UITextFieldDelegate
+class ColorsViewController :
+    UIViewController,
+    UICollectionViewDataSource,
+    UITextFieldDelegate
 {
-
-    class constants {
-        static let DFLTNUMCOLORS:Int=12
-    }
-
+    
     @IBOutlet weak var collectionView:UICollectionView!
     @IBOutlet weak var colorCountTextField: UITextField!
 
-    var ct: Int? = 12
+    var colorCount = 12
 
     override func viewDidLoad() {
         super.viewDidLoad()
-        self.collectionView.register(UINib(nibName: "ColorsCollectionViewCell", bundle: nil), forCellWithReuseIdentifier: "Cell")
-
+        self.collectionView.register(UINib(nibName: "ColorsCollectionViewCell",
+                                            bundle: nil),
+                        forCellWithReuseIdentifier: "Cell")
```

Comment:
Cell identifiers should be more descriptive than just "Cell". What if we wanted to modify the palette to include more than one type of cell?